src/app/data/pontusx-account-names.ts

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { Layer } from '../../oasis-nexus/api'
+import { Network } from '../../types/network'
+import { hasTextMatch } from '../components/HighlightedText/text-matching'
+import { AccountData, AccountEntry, AccountMap, AccountMetadataInfo } from './named-accounts'
+
+const DATA_SOURCE_URL = 'https://raw.githubusercontent.com/deltaDAO/mvg-portal/main/pontusxAddresses.json'
+
+const getPontusXAccountsMetadata = () =>
+  new Promise<AccountData>((resolve, reject) => {
+    axios.get(DATA_SOURCE_URL).then(response => {
+      if (response.status !== 200) reject("Couldn't load names")
+      if (!response.data) reject("Couldn't load names")
+      const map: AccountMap = new Map()
+      const list: AccountEntry[] = []
+      Object.entries(response.data).forEach(([address, name]) => {
+        map.set(address, { name: name as string })
+        const normalizedEntry: AccountEntry = {
+          name: name as string,
+          address,
+        }
+        list.push(normalizedEntry)
```

Comment:
I have done that now. (But I still don't want to unite the (trivial) lookup code, because it's entirely possible that some other layers will want an API-based lookup for addresses, so that must be kept separate.)